crates/bevy_ecs/src/event.rs

```diff
@@ -103,9 +107,11 @@ enum State {
 ///
 /// # Details
 ///
-/// [Events] is implemented using a double buffer. Each call to [Events::update] swaps buffers and
-/// clears out the oldest buffer. [EventReader]s that read at least once per update will never drop
-/// events. [EventReader]s that read once within two updates might still receive some events.
+/// [Events] is implemented using a variation of a double buffer strategy.
+/// Each call to [Events::update] swaps buffers and clears out the oldest one.
+/// [EventReader]s will read events from both buffers.
+/// [EventReader]s that read at least once per update will never drop events.
+/// [EventReader]s that read once within two updates might still receive some events.
```

Comment:
```suggestion
/// [`EventReader`]s that read once within two updates might still receive some events.
```

Please do this with all of the ones here, not just this one.

Reply:
I used MD as see fit in the scope of this PR, lmk if it's reasonable enough